operators/hive-operator/hive.package.yaml

```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: hive-operator.v1.1.16
+- currentCSV: hive-operator.v1.2.3179-0756b70e6
```

Comment:
So we're going with (re)starting the alpha channel at 1.2.COMMIT_NUM-HASH to match what we're doing for ocm channel?

Reply:
✓

Reply:
> So we're going with (re)starting the alpha channel at 1.2.COMMIT_NUM-HASH to match what we're doing for ocm channel?

The prerelease OCM channel (currently 2.4) will get the same bundles as `alpha` -- though the OCM channel is allowed to lag behind -- until we branch the next release. At that point we'll start naming the release bundles according to the OCM release, like 2.4.COMMIT_NUM-HASH, and the _new_ prerelease OCM channel will carry on following `alpha` with 1.2.x.